config/locales/zh_TW.yml

```diff
@@ -0,0 +1,40 @@
+zh_TW:
+  errors:
+    messages:
+      taken_in_past: '曾被使用過。'
```

Comment:
Not sure if we want the extra space at the end?

Reply:
Sorry if I'm wrong but I checked with different text editors and couldn't find the extra space?

Reply:
Oh interesting. It looks like there's space between it and the single quote at the end, but I guess that's just part of whatever character that's called? Anyway, verified through a text editor and there's no space character here so that's fine.